packages/idos-sdk-js/src/lib/kwil-wrapper.ts

```diff
@@ -2,12 +2,14 @@ import type { idOSUser, idOSUserAttribute, idOSWallet } from "@idos-network/idos
 import { KwilSigner, Utils as KwilUtils, WebKwil } from "@kwilteam/kwil-js";
 import type { ActionBody, ActionInput } from "@kwilteam/kwil-js/dist/core/action";
 import type { CustomSigner, EthSigner } from "@kwilteam/kwil-js/dist/core/builders.d";
+import Grant from "./grants/grant";
 
 export class KwilWrapper {
   static defaults = {
     kwilProvider: import.meta.env.VITE_IDOS_NODE_URL,
     chainId: import.meta.env.VITE_IDOS_NODE_KWIL_CHAIN_ID,
     dbId: import.meta.env.VITE_IDOS_NODE_KWIL_DB_ID,
+    grantsPerPage: 7,
```

Comment:
This is grants specific. `KwilWrapper.defaults` is way broader than that in scope. We need to find a better place to put it.